mintapi/api.py

```diff
@@ -151,7 +151,10 @@ def get_accounts(self, get_detail=False):  # {{{
 
     def get_transactions(self):
         if not pd:
-            raise ImportError('transactions data requires pandas; please pip install pandas')
+            raise ImportError(
+                'transactions data requires pandas; '
+                'please pip install pandas'
+            )
```

Comment:
I agree but [it wasn't PEP 8.](https://travis-ci.org/mrooney/mintapi/jobs/84168110)